includes/model/class-openedx-woocommerce-plugin-log.php

```diff
@@ -64,6 +67,26 @@ public function create_change_log( $post_id, $old_data_array, $enrollment_arr, $
 		}
 	}
 
+	/**
+	 * Check the API response from the Open edX API.
+	 *
+	 * @param array $response The API response.
+	 *
+	 * @return string $response The API response.
+	 */
+	public function check_api_response( $response ) {
+
+		if ( 'error' === $response[0] ) {
```

Comment:
Please avoid using many elseif statement since the code is harder to read provide a more clean solution using the switch like this : 
`public function check_api_response( $response ) {
        $actualResponse = $response[0];
        switch ($actualResponse) {
            case 'error':
                return $response[1];
                break;
            case 'success':
                return wp_json_encode( json_decode( $response[1], true ) );
                break;
            case 'not_api':
                return 'Enrollment saved locally, action does not require an API call';
                break;               
            default:
                return 'API did not provide a response';
                break;
        }
}`

Reply:
Done, thank you.